src/pages/Home/NavBar/SearchBox/SearchBoxStyle.ts

```diff
@@ -0,0 +1,15 @@
+import styled from "styled-components";
+
+export const SearchBoxStyle = styled.div`
```

Comment:
This should be an input. Otherwise the user can't type in the search

Reply:
I think search should be a separate page. This navigates to /search

Reply:
I don't think grab redirects the user to another page right?
why does it need to be in a separate page?